docs/sources/reference/components/prometheus.exporter.cloudwatch.md

```diff
@@ -90,8 +90,8 @@ To use all of the integration features, use the following AWS IAM Policy:
 
 ```alloy
 prometheus.exporter.cloudwatch "queues" {
-    sts_region = "us-east-2"
-
+    sts_region      = "us-east-2"
+    aws_sdk_version_v2 = "false"
     discovery {
         type        = "sqs"
```

Comment:
type=sqs is deprecated now - we need to search for all the examples everywhere and remove the usage of aliases

Reply:
I will update the tests. 👍 

We kept it to test and show the backward compatibility. 

A few commits ago, based on the discussion above, we added a compatibility layer to keep support for the aliases. 

- https://github.com/grafana/alloy/pull/839/commits/63ff123d9169a9bbbd4b5ae50a63ddb67f2e7483

The idea was to generate warnings but keep the support. 

@thampiotr, shall we use this approach or drop the support completely?

Reply:
I'm aware that there is compatibility and the old configs will work, my comment was more about using in our examples the official, correct names, not the aliases which are deprecated :)